modules/container_structure_test/1.15.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/GoogleContainerTools/container-structure-test/archive/e5848342114f9a435e9a54f45bef99a7703c9173.zip",
```

Comment:
note, after the next release we should be able to make this a normal, stable URL